eclipse-newsletter-dec-2018/README.md

```diff
@@ -2,42 +2,42 @@
 
```

Comment:
I may have messed up the line endings

Reply:
The audience is likely developers, so fixing EO jargon is probably necessary. Maybe next review.